test/hotspot/jtreg/ProblemList.txt

```diff
@@ -53,6 +53,7 @@ compiler/loopopts/TestUnreachableInnerLoop.java 8288981 linux-s390x
 
 compiler/c2/Test8004741.java 8235801 generic-all
 compiler/c2/irTests/TestDuplicateBackedge.java 8318904 generic-all
+compiler/c2/irTests/TestVectorizationMismatchedAccess.java 8342489 generic-ppc64,linux-s390x
```

Comment:
Should this be `generic-s390x`, or are we doing `linux-s390x` instead all the time?

Reply:
I think we only have linux-s390x in OpenJDK and use this everywhere.

Reply:
All right then.